frontend/src/hooks/background-hooks.ts

```diff
@@ -0,0 +1,17 @@
+import React, { useEffect } from "react";
+
+export function ChangeBackgroundColor(): any {
```

Comment:
By convention, our hooks start with "use" in lowercase like "useChangeBackgroundColor"